packages/venia-concept/src/components/App/__tests__/__snapshots__/app.spec.js.snap

```diff
@@ -390,13 +391,145 @@ ReactTestInstance {
                       "loc": "1",
                     },
                   ],
-                  [Function],
                 ],
                 "destroy": undefined,
-                "next": [Circular],
+                "next": Object {
```

Comment:
No problem here, but just wanted to document that we suspect the diff in all these React internals is because of moving all effects above any renders in `app.js`. To be safe, I'm requested @dpatil-magento does a full sanity check to make sure this didn't introduce any unexpected behavior.